packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -275,7 +266,7 @@ class AnsiTerminal {
     _INVALID_HANDLE,
   ];
 
-  bool supportsColor;
+  bool supportsColor = platform.stdoutSupportsAnsi;
```

Comment:
In package:platform: https://github.com/google/platform.dart/blob/master/lib/src/interface/local_platform.dart#L54